pom.xml

```diff
@@ -850,17 +849,17 @@
 				<plugin>
 					<groupId>net.revelc.code.formatter</groupId>
 					<artifactId>formatter-maven-plugin</artifactId>
-					<version>2.12.0</version>
+					<version>2.21.0</version>
 					<configuration>
-						<configFile>${maven.multiModuleProjectDirectory}/eclipse-settings/eclipse-rdf4j-conventions.xml</configFile>
+						<configFile>${project.basedir}/eclipse-settings/eclipse-rdf4j-conventions.xml</configFile>
 						<lineEnding>LF</lineEnding>
 						<skipHtmlFormatting>true</skipHtmlFormatting>
 					</configuration>
 				</plugin>
 				<plugin>
 					<groupId>net.revelc.code</groupId>
 					<artifactId>impsort-maven-plugin</artifactId>
-					<version>1.3.2</version>
+					<version>1.8.0</version>
 					<configuration>
```

Comment:
Also updated these.

And also changed the `maven.multiModuleProjectDirectory` to `project.basedir` because `maven.multiModuleProjectDirectory` is only for internal use. I don't think they exactly do the same thing, so I'm not sure if this is something we want to do.

Reply:
@jeenbroekstra do you know the background for using `maven.multiModuleProjectDirectory` and if changing this will be problematic?